datasets/ar_sarcasm/README.md

```diff
@@ -0,0 +1,156 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- Ibrahim Abu Farha
```

Comment:
The language_creators tags must be one of
```
- found
- expert-generated
- crowdsourced
- machine-generated
- other-<insert custom language creator tag in case the above is not relevant>
```
I guess you should use 
```
- found
```

since the texts come from other datasets